### PR TITLE
format code by black and isort

### PR DESCRIPTION
**Pull request by junnplus**
## What type of PR is this? 

Formating all files, and add black check in CI lint.

- [x] Refactor
- [ ] Feature
- [ ] Bug Fix
- [ ] New Query Runner (Data Source) 
- [ ] New Alert Destination
- [ ] Other

## How is this tested?

- [ ] Unit tests (pytest, jest)
- [ ] E2E Tests (Cypress)
- [ ] Manually
- [ ] N/A

### CODE REVIEW

**Comment by junnplus** [comment]
> Run black --check .
All done! ✨ 🍰 ✨
230 files would be left unchanged.

**Comment by justinclift** [comment]
Oh, this seems like a sensible idea. :smile:

File: setup.cfg [hunk 1]
```diff
@@ -2,6 +2,9 @@
...
 [flake8]
-ignore = E501
-exclude = .git
-max-complexity = 10
+ignore = E501, W503
+extend-ignore = E203
+exclude = .git,viz-lib,node_modules,migrations
+max-complexity = 15
+per-file-ignores =
+    __init__.py: F401
```
Comment (junnplus) [inline setup.cfg:10]: updated flake8 config.

File: redash/authentication/__init__.py [hunk 2]
```diff
@@ -229,7 +223,7 @@ def redirect_to_login():
 def logout_and_redirect_to_index():
     logout_user()
 
-    if settings.MULTI_ORG and current_org == None:
+    if settings.MULTI_ORG and current_org is None:
```
Comment (gaecoli) [inline redash/authentication/__init__.py:226]: I don't know if this needs to be changed. Maybe we can keep `==`.

**Review by gaecoli — APPROVED** [review]
LGTM!

File: redash/authentication/__init__.py (hunk 2 above)
Comment (junnplus) [reply redash/authentication/__init__.py]: OK, let's keep it as it is

File: bin/release_manager.py [hunk 3]
```diff
@@ -39,84 +44,96 @@ def get_rc_release(version):
 
     raise exception_from_error("Unknown error while looking RC release: ", response)
 
+
 def create_release(version, commit_sha):
     tag = rc_tag_name(version)
 
-    params = {
```
Comment (gaecoli) [inline bin/release_manager.py:45]: I think this style is better than your changed style.

File: redash/authentication/__init__.py (hunk 2 above)
Comment (gaecoli) [reply redash/authentication/__init__.py]: > OK, let's keep it as it is

OK.

File: bin/release_manager.py (hunk 3 above)
Comment (gaecoli) [reply bin/release_manager.py:45]: For readability, I think it`s better not to expand the code here.

Comment (junnplus) [reply bin/release_manager.py:45]: I might need to look into the configuration of black.

**Comment by guidopetri** [comment]
I'm not the biggest fan of `black`, I think it makes code pretty unreadable.

That being said, we should add this to a precommit file as well, to make it easy to keep the formatting inplace.

**Comment by junnplus** [comment]
@gaecoli Do you mind helping to review this PR again? There are quite a few changes in the code, so it might take some time.

**Comment by junnplus** [comment]
> That being said, we should add this to a precommit file as well, to make it easy to keep the formatting inplace.

Yes, this is my intention.

**Comment by gaecoli** [comment]
> @gaecoli Do you mind helping to review this PR again? There are quite a few changes in the code, so it might take some time.

I think it`s ok. I see that the code logic has not changed and the code style has been refined.
LGTM! Good job!

**Comment by gaecoli** [comment]
Look forward to your wiki. For ours how to use your tools to format code style.

**Pull request merged**